elastic/logs/templates/component/track-shared-logsdb-mode.json

```diff
@@ -1,21 +1,12 @@
 {
     "template": {
       "settings": {
+        {% if index_mode %}
         "index": {
-          {% if index_sorting == "hostname" %}
+            "mode": {{ index_mode | tojson }},
```

Comment:
I am not sure if there is any missing value for the hostname...just in case maybe we could add also `index.sort.missing`?